src/Microsoft.Identity.Client/MigrationAid/MigrationAid.cs

```diff
@@ -244,12 +245,35 @@ public partial class TokenCacheNotificationArgs
         public IUser User { get { throw new NotImplementedException(); } }
     }
 
+    public partial interface IPublicClientApplication
+    {
+#if WINDOWS_APP
+        /// <summary>
+        /// Flag to enable authentication with the user currently logeed-in in Windows.
+        /// When set to true, the application will try to connect to the corporate network using windows integrated authentication.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        [Obsolete("PublicClientApplication is now immutable, you can set this property using the PublicClientApplicationBuilder and read it using IAppConfig.  See https://aka.ms/msal-net-application-configuration", true)]
```

Comment:
See https://aka.ms/msal-net-3-breaking-changes and https://aka.ms/msal-net-application-configuration